packages/runtime/native/shared/Css_types.ml

```diff
@@ -3941,13 +3941,43 @@ module FontFace = struct
 end
 
 module FontFamilyName = struct
-  type t = string
+  type t =
+    [ `serif
+    | `sans_serif
+    | `monospace
+    | `cursive
+    | `fantasy
+    | `system_ui
+    | `ui_serif
+    | `ui_sans_serif
+    | `ui_monospace
+    | `ui_rounded
+    | `emoji
+    | `math
+    | `fangsong
+    | `custom of string
```

Comment:
Did you thought about another name rather than "custom"? 
`quoted` or `string` comes to mind and feels cleaner. This is rather a nitpick

Reply:
`named`  or `specific` came to mind but will stick to `quoted`